doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run `VxWorks`_ as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
+NUC (model NUC7i5DNHE) in this tutorial. Other :ref:`ACRN supported platforms <hardware>` should work as well.
+
```

Comment:
As a proprietary OS,  you need to buy a copy of VxWorks, right?  You should mention this as a prerequisite of this tutorial.  Maybe this:
```
.. note:: You'll need to be a WindRiver* customer and have purchased VxWorks to follow this tutorial.
```

Reply:
Will add it in next version.